src/commands/play.rs

```diff
@@ -76,7 +76,7 @@ pub async fn play(
     let manager = songbird::get(ctx).await.unwrap();
 
     // try to join a voice channel if not in one just yet
-    summon(ctx, interaction, false).await?;
+    summon(ctx, interaction, false, false).await?;
```

Comment:
Why always false? If this is the first command since launch, settings should be loaded.
Regardless, tying the loading of the settings to the `summon` command isn't the best solution.